code/__DEFINES/text.dm

```diff
@@ -7,3 +7,5 @@
 #define CENTER(text) {"<center>[##text]</center>"}
 
 #define SANITIZE_FILENAME(text) (GLOB.filename_forbidden_chars.Replace(text, ""))
+
```

Comment:
Why?

Reply:
i added a macro bc i didnt know how pick() worked, just ignore

Reply:
remove the newlines